src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -1627,12 +1646,13 @@ internal string ToStringWithoutPassword()
             return clone.ToString();
         }
 
-        internal string ConnectionStringWithoutTargetSessionAttributes
+        internal string ConnectionStringForMultipleHosts
         {
             get
             {
                 var clone = Clone();
                 clone[nameof(TargetSessionAttributes)] = null;
+                clone[nameof(LoadBalanceHosts)] = null;
```

Comment:
My reasoning was to exclude it because `LoadBalanceHosts` only makes sense with multiple hosts. But as I do not have a valid example on when it might be useful, I have nothing against of reverting this change.